pkg/k8shandler/elasticsearch_helper.go

```diff
@@ -101,20 +101,24 @@ func updateAllIndexReplicas(clusterName, namespace string, client client.Client,
 
 	// get list of indices and call updateIndexReplicas for each one
 	for index, health := range indexHealth {
-		// only update replicas for indices that don't have same replica count
-		if numberOfReplicas := parseString("settings.index.number_of_replicas", health.(map[string]interface{})); numberOfReplicas != "" {
-			currentReplicas, err := strconv.ParseInt(numberOfReplicas, 10, 32)
-			if err != nil {
-				return false, err
-			}
+		if _, ok := health.(map[string]interface{}); ok {
```

Comment:
Tangential but maybe we need to consider replacing this with a proper elasticsearch client that may already handle these subtleties